### PR TITLE
add script and instruction for protoc plugins debug

### PR DESCRIPTION
**Pull request by burov**
I provide some kind of tutorial and utill scripts for protoc plugins debugging

### CODE REVIEW

**Comment by Evgeniy-L** [comment]
I remember we discussed if this need to go to toolkit, [gentool](https://github.com/infobloxopen/atlas-gentool) or a separate package. I think we had idea of having it in [gentool](https://github.com/infobloxopen/atlas-gentool)

**Comment by burov** [comment]
@dkukharau We want review from you

File: tools/debugging/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.10.0 AS builder
```
Comment (dkukharau) [inline tools/debugging/Dockerfile]: Why have this file here if we've already put it in README?

Comment (burov) [reply tools/debugging/Dockerfile]: Because we use this file in ```docker build``` command, you must change it and use as a default behavior

**Pull request merged**